### PR TITLE
Integrate Styled components in Preview Chat in BotBuilder #2442

### LINKED ISSUE #2442: Integrate Styled components in Preview Chat in BotBuilder
**Actual Behaviour**

CSS being used

**Expected Behaviour**

Integrate Styled components in Preview Chat in BotBuilder

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2442

Changes: 
- Integrate Styled components in Preview Chat in BotBuilder
- CSS Cleanup

Demo Link: https://pr-2472-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Desktop View:

Screenshot 2019-07-07 at 12 25 26 AM

Mobile View:

Screenshot 2019-07-07 at 12 25 39 AM

### CODE REVIEW

**Comment by fragm3** [comment]
> Why are you not completing deleting the css file?

That's because it was in Work in Progress.

**Comment by fragm3** [comment]
**Note**: Some classnames are required for `ChatApp.css` to inject styles

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 1]
```diff
@@ -2,12 +2,182 @@ import React, { Component } from 'react';
...
+  background-size: 60px;
+  text-align: right;
+  float: right;
+  margin-right: 0%;
+  margin-top: 15px;
+  font-family: Open Sans, sans-serif;
+  border-radius: 5em;
+  cursor: pointer;
+  transition: transform .15s ease-in-out, box-shadow .15s ease-in-out;
+  visibility: visible;
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: Not required.

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 2]
```diff
@@ -2,12 +2,182 @@ import React, { Component } from 'react';
...
+`;
+
+const SUSILauncherWrapper = styled.div`
+  width: 60px;
+  background-size: 60px;
+  text-align: right;
+  float: right;
+  margin-right: 0%;
+  margin-top: 15px;
+  font-family: Open Sans, sans-serif;
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: not required

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 3]
```diff
@@ -2,12 +2,182 @@ import React, { Component } from 'react';
...
+  padding: 0;
+  height: auto;
+  :hover {
+    visibility: visible;
+    box-shadow: 0 4px 42px 0 rgba(0, 0, 0, .25);
+  }
+`;
+
+const SUSILauncherButton = styled.div`
+  background-color: ${props => props.backgroundColor};
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: not required

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 4]
```diff
@@ -2,12 +2,182 @@ import React, { Component } from 'react';
...
+  width: 60px;
+  height: 60px;
+  background-size: 60px;
+  border-radius: 50%;
+  margin: -1px;
+  box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.15);
+  right: 0;
+  background-position: 50%;
+  background-repeat: no-repeat;
+  visibility: visible;
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: not required

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 5]
```diff
@@ -194,118 +347,89 @@ class Preview extends Component {
...
+                      </SUSIMessageContainer>
+                    </div>
+                    <div className="susi-composer-container">
+                      <div id="susi-composer" className="susi-composer ">
+                        <div className="susi-composer-textarea-container">
+                          <div className="susi-composer-textarea">
+                            <SendButtonWrapper>
+                              <Send
+                                onClick={this.sendMessage}
+                                style={{
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: Use styled component

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 6]
```diff
@@ -194,118 +347,89 @@ class Preview extends Component {
...
                     </div>
                   </div>
                 </div>
-              </div>
-              {botBuilder ? (
-                <div
-                  id="susi-launcher-close"
-                  title="Close"
+                <CloseButton
+                  style={{ fill: '#fff' }}
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: use styled component

File: src/components/cms/BotBuilder/Preview/Preview.js (hunk 3 above)
Comment (fragm3) [reply src/components/cms/BotBuilder/Preview/Preview.js]: This is required as we have Avatar background option in BotBuilder --> Design --> Avatar background

**Comment by fragm3** [comment]
@harsh-jindal made the changes

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 7]
```diff
@@ -2,12 +2,185 @@ import React, { Component } from 'react';
...
+  height: 30px;
+  background-color: #666;
+  border-radius: 50%;
+  fill: #fff;
+`;
+
+const SendButtonWrapper = styled.pre`
+  margin: 0;
+  cursor: pointer;
+  font-family: Open Sans, sans-serif;
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: not required

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 8]
```diff
@@ -2,12 +2,185 @@ import React, { Component } from 'react';
...
+  margin: 0;
+  border-radius: 8px;
+  box-shadow: 0 5px 40px rgba(0, 0, 0, 0.16);
+  overflow-x: hidden;
+  height: 460px;
+  width: 100%;
+  animation: ${moveFromBottomFadeKeyframe} 0.3s ease both;
+
+  media(max-width: 667px) {
+    width: 100%;
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: not required

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 9]
```diff
@@ -2,12 +2,185 @@ import React, { Component } from 'react';
...
+  margin-top: 15px;
+  border-radius: 5em;
+  cursor: pointer;
+  transition: transform .15s ease-in-out, box-shadow .15s ease-in-out;
+  transform: translateY(150px);
+  animation: ${launcherFrameAppearKeyframe} .25s ease forwards
+  padding: 0;
+  height: auto;
+  :hover {
+    visibility: visible;
```
Comment (harsh-jindal) [inline src/components/cms/BotBuilder/Preview/Preview.js]: not required

**Comment by harsh-jindal** [comment]
@fragm3 Just make these final changes. Rest LGTM

**Comment by fragm3** [comment]
@harsh-jindal made the changes

**Pull request merged**